CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 
 ### Removed
 
+- Individual etcd member's services were removed. Use hostname and subdomain of etcd pod instead.
```

Comment:
@xiang90 Just a quick question: 
Are we creating a `<pod-subdomain>` headless service anywhere so that each etcd pod is accessible by `http://<pod-hostname>.<pod-subdomain>.<namespace>.cluster.local`?